android/fastlane/metadata/android/en-GB/changelogs/10041.txt

```diff
@@ -0,0 +1,2 @@
+Added:
+     -Users can now choose from setting if they want to see in the week indicator,from the timetable,the academic week number.
```

Comment:
Keep in mind that this is addressed to users. Use the other changelogs as examples. This is as if I was talking to you and saying "Andrei can now pass the maths class". It's usually also good to keep it short, and be consistent with the previous formatting. Furthermore, with every release we need to include all the changes that happened since the last release (this you can usually see from the commit history).

```suggestion
Added
  - The option to show the number of the week in the semester instead of the week in the year.
  - A feature to allow admins to send a confirmation email from the app when approving a permission request.

Fixed
  - Some minor bugs and text issues.
```

Try to do the same thing in the other changelogs as well.

(and also, you'll need to rename them since the version was bumped since the last time you created these files)